browser/main/modals/PreferencesModal/StoragesTab.js

```diff
@@ -57,8 +91,61 @@ class StoragesTab extends React.Component {
     e.preventDefault()
   }
 
+  removeAllAttachments (attachments) {
```

Comment:
Maybe this method should also be in the attachmentManagement-file..?

Reply:
And a test for it is required

Reply:
If I'm gonna put it in attachmentManagement, maybe I should change the `getAttachments` function return types to compatible with this. Well I plan to do that anyway, let's do it.